.travis.yml

```diff
@@ -10,6 +10,7 @@ branches:
     - master
     - beta
     - release
+    - next
```

Comment:
Probably can remove?

Reply:
Though I guess it doesn't matter much...

Reply:
will remove it once this PR is merged. otherwise PRs against `next` will not run CI.